lib/kontena/plugin/cloud/node/create_command.rb

```diff
@@ -79,11 +79,7 @@ def default_count
   end
 
   def current_platform
-    if ENV['KONTENA_PLATFORM']
-      ENV['KONTENA_PLATFORM']
-    else
-      current_master.name
-    end
+    ENV['KONTENA_PLATFORM'] || current_master.name
```

Comment:
This type of defaulting does not take into consideration that the env variable could be an empty string.
Perhaps generally it would be better  to do something like:

```ruby
(!ENV['FOOFOO'].to_s.empty? && ENV['FOOFOO']) || 'default'
```

The same pattern is repeated everywhere so it's not a problem for this PR. Just making a note.